Formula/varnish.rb

```diff
@@ -22,7 +21,7 @@ class Varnish < Formula
   depends_on "docutils" => :build
   depends_on "graphviz" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.9" => :build
+  depends_on "python@3.10" => :build
```

Comment:
I don't see an update for the Python version in `install`:

```ruby

ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"

```

That'll need updating too.

Reply:
#89929